### PR TITLE
Feature/6 splash screen UI

### LINKED ISSUE #6: Splash screen UI
# REQUIREMENTS
Develop the splash screen
# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.

Screen Shot 2020-12-03 at 11 34 11

# Acceptance Criteria
List all required acceptance criteria
- [ ] Splash screens show correctly

### PR DESCRIPTION
**Pull request by Ocosmee**
# Description
This PR is linked to issue #6 
 - Create  splash screen design

 # Testing
- npm-install to download packages
- npx react-native start.
- npx react-native run-android.
 
image 2020-12-09 at 12 11 39 PM

  
  ## Screenshots 
- Omit the watermark
splash

### CODE REVIEW

**Review by tovl1 — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by JordiGN — DISMISSED** [review]

File: tribomobile/screens/splash.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  StyleSheet,
+  Animated,
+  Dimensions,
+  StatusBar,
+} from 'react-native';
+
+function SplashScreen () {
+  const logoAnim = useRef(new Animated.Value(0)).current;
+  const moveAnim = useRef(new Animated.Value(0)).current;
+  const fadeAnim = useRef(new Animated.Value(0)).current;
```
Comment (fernandoamz) [inline tribomobile/screens/splash.js]: Try don't repeat methods

File: tribomobile/screens/splash.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <StatusBar barStyle="dark-content" hidden = {true} />
+      <View style={styles.contentContainer}>
+        <Animated.Image
+          style={[styles.logoSmall,{opacity: fadeAnim}]}
+          source={require('../assets/mountain.png')}
+        />
+        <Animated.View style={[styles.logoContainer, {marginLeft: moveAnim}]}>
+          <Animated.Text style={[styles.logoText]}>L</Animated.Text>
+            <Animated.Text style={[styles.logoText, {opacity: fadeAnim}]}>
+              AMARKET.APP
```
Comment (fernandoamz) [inline tribomobile/screens/splash.js]: Try to add the texts in constants file

File: tribomobile/screens/splash.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            </Animated.Text>
+        </Animated.View>
+        <Animated.Image
+          style={[styles.logoBig,{ opacity: fadeAnim}]}
+          source={require('../assets/mountainbig.png')}
+        />
+      </View>
+      <View style={{justifyContent: 'space-between', alignItems: 'center', marginBottom: 20}}>
+        <Animated.Text style={[styles.titleEnd, {opacity: fadeAnim} ]}>Made with love in Colima, México</Animated.Text>
+        <Animated.Text style={[styles.titleEnd, {opacity: fadeAnim} ]}>#WeAreMagma</Animated.Text>
```
Comment (fernandoamz) [inline tribomobile/screens/splash.js]: same case, Add to constant file

File: tribomobile/screens/splash.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#F1EFEF',
```
Comment (fernandoamz) [inline tribomobile/screens/splash.js]: Standardize the colors into a constants file

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: tribomobile/App.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  StatusBar,
+  View,
+} from 'react-native';
+import SplashScreen from './screens/splash'
+
+const App: () => React$Node = () => {
+  return (
+    <>
+        <SplashScreen/>
+    </>
```
Comment (luisarmandodv) [inline tribomobile/App.js]: I suggest using 2 spaces for indentation instead 4 spaces.
```suggestion
    <>
      <SplashScreen/>
    </>
```

File: tribomobile/App.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+import React from 'react';
+import {
+  SafeAreaView,
+  Text,
+  StatusBar,
+  View,
+} from 'react-native';
+import SplashScreen from './screens/splash'
```
Comment (luisarmandodv) [inline tribomobile/App.js]: file name `splash.js` should be `Splash.js` in upper-case

**Review by luisarmandodv — CHANGES_REQUESTED** [review]
it seems there are animated components. Could you share a gif in your PR to see how it works?

File: tribomobile/screens/splash.js [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+    width: 70, 
+    height: 35, 
+  },
+  logoContainer: {
+    flexDirection: 'row',
+  },
+  logoText: {
+    fontSize: 20,
+    marginTop: 5,
+    color: '#4E4D4D',
```
Comment (LeyaimMagma) [inline tribomobile/screens/splash.js]: Add to colors file

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1 @@
+tribomobile/node_modules
```
Comment (LeyaimMagma) [inline .gitignore]: Add blank space

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by JordiGN — DISMISSED** [review]
Good job, just a reminder, before commit your changes and remember to use prettier, great job with the splash image by the way!

File: tribomobile/screens/splash.js (hunk 7 above)
Comment (Ocosmee) [reply tribomobile/screens/splash.js]: The global colors are a different branch with a PR no approved yet

File: tribomobile/App.js [hunk 9]
```diff
@@ -7,34 +7,21 @@
...
-import {
-  SafeAreaView,
-  Text,
-  StatusBar,
-  View,
-} from 'react-native';
-import titlesText from './src/titlesText';
 import ModalMyAccount from './components/modals/ModalMyAccount';
-
 import {SafeAreaView, Text, StatusBar} from 'react-native';
```
Comment (1ng-HeberH) [inline tribomobile/App.js:11]: do you have two import calls?

File: tribomobile/App.js [hunk 10]
```diff
@@ -7,34 +7,21 @@
...
 import ModalInfoStore from './components/modals/ModalInfoStore';
 import LittlePin from './components/modals/LittlePinInfo';
+import titlesText from './src/titlesText';
+import contentText from './screensText/contentText';
```
Comment (luisarmandodv) [inline tribomobile/App.js]: good practices says we must use upper-case naming format for all components.

```suggestion
import TitlesText from './src/TitlesText';
import ContentText from './screensText/ContentText';
```

**Review by luisarmandodv — CHANGES_REQUESTED** [review]

**Review by JordiGN — DISMISSED** [review]
Excellent, the animation looks cool!

**Review by Ing-Heber — DISMISSED** [review]
Nice job Cosme

**Review by tovl1 — DISMISSED** [review]

**Review by JordiGN — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Comment by Ing-Heber** [comment]
Niceeee!!

**Review by luisarmandodv — DISMISSED** [review]

**Pull request merged**